src/containers/BooksForm.jsx

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { createBook } from '../actions';
+
+const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+
+class BookForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { title: '', category: 'Action' };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleSubmit(e) {
+    e.preventDefault();
+    const { props: { createBook }, state: { title, category } } = this;
+    const book = {
+      id: Math.random().toString(),
+      title,
+      category,
+    };
+    createBook(book);
+    this.setState({ title: '' });
+  }
+
+  handleChange(event) {
+    this.setState({ [event.target.name]: event.target.value });
+  }
```

Comment:
[Optional]

Why not destructure directly? Seems to me that you not using `event` or `target` in the `handleChange` function... 🤔 

We could have something like this instead:
```suggestion
  handleChange({target: {name, value}}) {
    this.setState({ [name]: value });
  }
```